jquery.canvaswrapper.js

```diff
@@ -344,24 +302,27 @@
         // If we can't find a matching element in our cache, create a new one
 
         if (!info) {
-            var element = document.createElement('div');
-            element.innerHTML = text;
+            var element = document.createElementNS("http://www.w3.org/2000/svg", "text");
+            var textNode = document.createTextNode(text);
+            element.appendChild(textNode);
             element.style.position = 'absolute';
             element.style.maxWidth = width;
-            element.style.top = '-9999px';
+            element.setAttributeNS(null, "x", -9999);
+            element.setAttributeNS(null, "y", -9999);
 
             if (typeof font === 'object') {
                 element.style.font = textStyle;
-                element.style.color = font.color;
+                element.style.fill = font.fill;
             } else if (typeof font === 'string') {
-                element.className = font;
+                element.className.baseVal = font;
```

Comment:
If possible maybe it's better to switch to a more widely used method to set the class to an SVG element:
element.setAttribute('class', font);